### ISSUE TITLE
MacOS

### ISSUE THREAD

**Issue by benn44** [REVENGE977/stremio-enhanced-community#8]
How can I download Stremio Enhanced on MacOS (M-series)

**Comment by Dyras** [comment]
Download the ARM64 version?

https://github.com/REVENGE977/stremio-enhanced-community/releases/download/v0.6/stremio-enhanced-darwin-arm64.zip

**Comment by olivergiangvu** [comment]
> Download the ARM64 version?
> 
> https://github.com/REVENGE977/stremio-enhanced-community/releases/download/v0.6/stremio-enhanced-darwin-arm64.zip

Downloaded but cant run on macOS 15

**Comment by dami2106** [comment]
Yep, does not run on MacOS 15 (24A335). Gives error `The application “stremio-enhanced" can’t be opened.` when trying to open the downloaded application.

**Comment by Vegals** [comment]
Bumping this. "This app can't be opened" message. M3, Sequoia

**Comment by Vegals** [comment]
> > Download the ARM64 version?
> > https://github.com/REVENGE977/stremio-enhanced-community/releases/download/v0.6/stremio-enhanced-darwin-arm64.zip
> 
> Downloaded but cant run on macOS 15

Actually, I figured out why, might work for you. 
I was downloading from safari, the file would auto decompress the .zip file which messed it up. Ended up downloading it on free download manager, chrome would have been fine too. Then using unarchiver to extract the files. 

Next issue now is when I open I get an error Stemio service not found please download, and I have it download. haha. All well, step closer.

**Comment by dfols** [comment]
@Vegals i'm getting the same "Stremio service not found" error on mac

**Comment by Vegals** [comment]
> @Vegals i'm getting the same "Stremio service not found" error on mac

Ah sorry to hear. I ended up trying to go in and fix it for Mac but wasn't able to suss it. Not smart enough.

**Comment by REVENGE977** [comment]
This should no longer be an issue in the latest release if you're on an x86 Mac (I tested it earlier on a macOS VM). However, if you're on an ARM-based Mac (M1/M2), I believe this might still be an issue.

I plan to switch to using GitHub Actions in upcoming updates and build the project with electron-builder. That might resolve the issue.

Right now, I'm using electron-packager, which isn’t ideal for cross-platform compatibility.

**Comment by thegy55555** [comment]
yeah, still doesn't work on M1 Mac as you predicted, sorry

**Comment by REVENGE977** [comment]
> yeah, still doesn't work on M1 Mac as you predicted, sorry


Try these out, see if either works on your M1:

[Stremio.Enhanced-0.9.1-arm64.dmg](https://github.com/REVENGE977/stremio-enhanced/releases/download/v20250413182358/Stremio.Enhanced-0.9.1-arm64.dmg)

[mac-arm64.zip](https://github.com/REVENGE977/stremio-enhanced/releases/download/v20250413182358/mac-arm64.zip
)

**Comment by REVENGE977** [comment]
I forgot an important note. The app isn't signed so if you're on macOS you'll have to bypass Gatekeeper.

You should right-click the app and choose “Open” for first time you open the app or run on terminal this command:

```sh
xattr -cr /path/to/Stremio.Enhanced.app
```

**Comment by Codexivus** [comment]
Followed Vegals advice and had the same error unfortunately. 

Process was:

- Download arm64 9.2, installed but The application “stremio-enhanced” can’t be opened appeared
- Removed and  then downloaded mac-arm64.zip, not able to install, but able do boot the app directly. Encountered same error about missing Stremio service. 
-Tried installing arm64 9.2 again and got Stremio Enhanced” is damaged and can’t be opened. You should eject the disk image.

**Comment by computechnopro** [comment]
### **Solution: -**
Go to the terminal and type the following.
`xattr -cr '/Applications/Stremio Enhanced.app'`
Press enter, and you're done! Run the app now.
